igvm/hypervisor_preferences.py

```diff
@@ -12,30 +12,43 @@
 # them reusable, even though most of them are not reused.  Some of the classes
 # are so simple that they could as well just be a function, but kept
 # as classes to have a consistent style.
-
+import math
+import sys
 from logging import getLogger
-
-from igvm.utils import LazyCompare
+from typing import Union, List, Optional
 
 log = getLogger(__name__)
 
 
-class InsufficientResource(object):
+class HypervisorPreference(object):
```

Comment:
Passing `object` is optional in Python3. If not specified everything implicitly inherits from object.

Reply:
true. fixed